dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/enrollment/EnrollmentRequestParamsMapper.java

```diff
@@ -69,7 +69,7 @@ public EnrollmentOperationParams map(RequestParams requestParams) throws BadRequ
         validateDeprecatedParameter(
             "ouMode", requestParams.getOuMode(), "orgUnitMode", requestParams.getOrgUnitMode());
 
-    validateOrgUnitParams(orgUnits, orgUnitMode);
+    validateOrgUnitMode(orgUnits, orgUnitMode);
```

Comment:
Yes, but that's for enrollments. 
In this PR I'm focusing on tracked entities. I'd rather do that when we work on that endpoint, to avoid making unnecessary mistakes.

Reply:
You did make the change for events though 🤔 even though the PR description says trackedEntities 🤷🏻

Reply:
Well yes, but the validations are the same.
What I don't want to do is change the value assigned to the `orgUnitMode`, because that could lead to errors I don't want to deal with in this PR.